code/drasil-build/lib/Build/Drasil/Make/AST.hs

```diff
@@ -29,12 +31,12 @@ type Dependencies = [Target]
 -- * Constructors
 
 -- | Creates a Rule which results in a file being created.
-mkFile :: Target -> Dependencies -> [Command] -> Rule
-mkFile t d = R t d File
+mkFile :: [Comment] -> Target -> Dependencies -> [Command] -> Rule
```

Comment:
It might be nice to add a `type Annotation = [Comment]` and use it here. What do you think?

Reply:
Good idea.

Reply:
I will make that change right now!